yarn.lock

```diff
@@ -4342,15 +4753,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"randombytes@npm:^2.1.0":
-  version: 2.1.0
-  resolution: "randombytes@npm:2.1.0"
-  dependencies:
-    safe-buffer: ^5.1.0
-  checksum: d779499376bd4cbb435ef3ab9a957006c8682f343f14089ed5f27764e4645114196e75b7f6abf1cbd84fd247c0cb0651698444df8c9bf30e62120fbbc52269d6
-  languageName: node
-  linkType: hard
-
```

Comment:
seems like [yarn 3.3.1 release](https://github.com/MetaMask/eth-trezor-keyring/blob/main/.yarn/releases/yarn-3.3.1.cjs) relies on `randombytes`...  but doesn't implement it or import it itself so appears to rely on it being in our dependency tree elsewhere... can this be right @Gudahtt?

Reply:
hence this error: https://github.com/MetaMask/eth-trezor-keyring/actions/runs/3904473415/jobs/6670319743

Reply:
Hmm. Agreed that that doesn't sound right. Even if they did rely on that, `randombytes` is still in our dependency tree.

Reply:
Ah, it seems to be an undeclared dependency in the version of `trezor-connect` that we're using

Reply:
@mikesposito I started to investigate this but got sidetracked. Basically `trezor-connect` no longer recommends using sub 9.0.0 versions and have marked other versions as deprecated. I started to attempt the migration up to version 9, but it was a bit trickier than I hoped. We should do that as a pre-requisite to completing this ticket.

Reply:
Actually a PR to update to v9 already exists: https://github.com/MetaMask/eth-trezor-keyring/pull/133.

Reply:
And it's merged!